dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/WebModuleOpenAPIProvider.java

```diff
@@ -144,4 +159,14 @@ public String getApplicationPath() {
     public boolean getServersDefined() {
         return serversDefined;
     }
+
+    @Override
+    public void waitForBackgroundTasks() {
```

Comment:
<bikeshedding>This is OK but the name might be better if it had a notion of there being timeout in it, or the timeout was passed in perhaps</bikeshedding>